Client/Frontend/SearchSettingsTableViewController.swift

```diff
@@ -0,0 +1,175 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+class SearchSettingsTableViewController: UITableViewController {
```

Comment:
On Thu, Feb 26, 2015 at 3:55 PM, wes johnston <notifications@github.com>
wrote:

> In Client/Frontend/SearchSettingsTableViewController.swift
> <https://github.com/mozilla/firefox-ios/pull/187#discussion_r25477860>:
>
> > @@ -0,0 +1,175 @@
> > +/* This Source Code Form is subject to the terms of the Mozilla Public
> > + * License, v. 2.0. If a copy of the MPL was not distributed with this
> > + * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
> > +
> > +import UIKit
> > +
> > +class SearchSettingsTableViewController: UITableViewController {
>
> Can we move this Settings stuff into a Settings folder (maybe it is in
> XCode's view?)
>

It is, but I'm happy to make it a folder on disk as well.